src/commands/android/index.ts

```diff
@@ -17,8 +17,10 @@ import {
 } from './constants';
 import {AndroidSetupResult, Options, OtherInfo, Platform, SdkBinary, SetupConfigs} from './interfaces';
 import {
+  checkJavaInstallation,
   downloadFirefoxAndroid, downloadWithProgressBar, getAllAvailableOptions,
-  getBinaryLocation, getBinaryNameForOS, getFirefoxApkName, getLatestVersion
+  getBinaryLocation, getBinaryNameForOS, getFirefoxApkName, getLatestVersion,
+  getSdkRootFromEnv
```

Comment:
Can you re-arrange these import statements? Move the last import from each line one line down.

Reply:
I am having a bit of trouble understanding this. Can you please show some example statement or something?

Reply:
What I meant is to change the above imports to the following:
```js
  checkJavaInstallation, downloadFirefoxAndroid, downloadWithProgressBar,
  getAllAvailableOptions, getBinaryLocation, getBinaryNameForOS,
  getFirefoxApkName, getLatestVersion, getSdkRootFromEnv
```
(instead of having a single import in the first and the last line, re-arrange the imports.